recipe-server/client/control_new/components/recipes/RecipeForm.js

```diff
@@ -155,3 +159,48 @@ class ActionSelect extends React.Component {
     );
   }
 }
+
+@connect(
+  state => ({
+    githubUrl: getGithubUrl(state),
+  })
+)
+class ArgumentEditorMissingError extends React.PureComponent {
+  static propTypes = {
+    githubUrl: PropTypes.string,
+    name: PropTypes.string.isRequired,
+  }
+
+  defaultPropTypes = {
+    githubUrl: null,
+  }
+
+  render() {
+    const { githubUrl, name } = this.props;
+    let fileIssueUrl;
+    if (githubUrl) {
+      const u = new URL(githubUrl);
```

Comment:
We should use more descriptive variable names. I'm surprised this wasn't hit by linting, I thought there was a 3 character minimum or something.